amplipi/streams.py

```diff
@@ -795,10 +795,28 @@ def send_cmd(self, cmd):
     """
     try:
       if cmd in self.supported_cmds:
+        if cmd == "love":
+          src_config_folder = f'{utils.get_folder("config")}/srcs/v{self.vsrc}'
+          with open(f'{src_config_folder}/.config/pianobar/currentSong', 'r+', encoding='utf-8') as file:
+            data = file.read()
+            chunks = data.split(",,,")
+            if len(chunks) >= 5:
```

Comment:
can you clarify in a comment what `len(chunks) >= 5` means in this context? there's a bunch of data parsing here but no description of the expected format or what you are attemping to scrape out. should someone else come along and have to debug this logic (for example, if the upstream format changes), this chunk of code is almost easier to scrap and rewrite than figure out what it intends to do.